grails-app/services/au/org/ala/merit/UserService.groovy

```diff
@@ -682,4 +682,16 @@ class UserService {
         return response?.doesUserHaveHubProjects
     }
 
+    /**
+     * Checks if a user have a role on an existing MERIT project.
```

Comment:
This comment isn't correct.
Fix or delete javadoc.
Consider renaming this "doesUserHavePermissionExpiringWithinAMonth"